hollow-diff-ui/build.gradle

```diff
@@ -9,12 +9,13 @@ sourceSets {
  
 dependencies {
     compile project(':hollow')
-    compile 'org.eclipse.jetty:jetty-server:9.3.0.M1'
     compile 'org.apache.velocity:velocity:1.7'
     compile 'commons-codec:commons-codec:1.8'
     compile 'commons-io:commons-io:2.3'
     compile 'com.google.code.gson:gson:2.8.0'
 
+    compileOnly 'org.eclipse.jetty:jetty-server:9.+'
```

Comment:
Since jetty likes to publish milestones and rc's to maven central, its probably worth noting that 9.+ like this will still get you milestones and rc's. You're not shipping them to your consumers, but it might cause you some headaches as you go.